pandas/io/tests/test_date_converters.py

```diff
@@ -138,6 +138,19 @@ def date_parser(date, time):
                                names=['datetime', 'prn']))
         assert_frame_equal(df, df_correct)
 
+    def test_parse_date_column_with_empty_string(self):
+        # GH 6428
+        data = """case,opdate
+                  7,10/18/2006
+                  7,10/18/2008
```

Comment:
You were correct; current tests require `errors='ignore'`. 

This is tricky since `errors='ignore'` and `errors='coerce'` both terminate the parse with `NaT`s or returning the input. Would it be possible to introduce a new kwarg (e.g. `date_errors`) to pass ignore/coerce/raise, into `to_datetime` within `read_csv`?